controller/proxy-injector/fake/data/pod-with-debug.patch.json

```diff
@@ -176,6 +176,14 @@
           "name": "LINKERD2_PROXY_INBOUND_PORTS_DISABLE_PROTOCOL_DETECTION",
           "value": "25,443,587,3306,4444,5432,6379,9300,11211"
         },
+        {
```

Comment:
We didn't need to add the other env variables in this patch because **identity is disabled** in these tests.

This raises the question, Should we set the new policy variables even when identity is disabled? 🤔

Reply:
yeah, I think we should set them with identity disabled...

Reply:
Updated!